packages/api/lib/writeRecords/write-granules.js

```diff
@@ -666,6 +690,8 @@ const writeGranuleRecordAndPublishSns = async ({
     knex,
     postgresGranuleRecord,
   });
+  const pgGranule = pgGranuleResult.pgGranule;
+
   await _publishPostgresGranuleUpdateToSns({
```

Comment:
I was unsure about this method, but it seemed possible that something needed to be adjusted for it, depending on the status of the update.

Reply:
@npauzenga I know we discussed this yesterday, but a bit more information on the method - in addition to another call down the publish sns topic chain, it logs `log.info('Successfully wrote granule %j to SNS topic', granuletoPublish);`.